### DISCUSSION TITLE
[firebase_messaging] Automatic launch app on receiving specific Notification

### DISCUSSION THREAD

**Discussion by prafgup** [firebase/flutterfire#5598] [Feature request]
There is currently no way to launch a specific UI screen on receiving specific notification.
eg. Showing Call pickup ui when received Video Call notification.

firebase_messaging whould have a specific function to register callbacks which launches app even when the app is closed.

**Comment by kratipandey57** [comment]
how to launch app on receiving  notification. any solution for this problem ?

**Comment by prafgup** [comment]
@kratipandey57 Sadly theres no solution for this yet.
